examples/effortless_audit/windows/Readme.md

```diff
@@ -0,0 +1,46 @@
+# Effortless Audit
```

Comment:
I don't think we need a duplicate readme for windows since this is similar to the readme that is already there I think it needs to be moved to the top level of the Effortless Audit examples folder.